types/object_store.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+const (
+	ObjectStoreContainerPort = 7480
+	ObjectStoreServicePort   = 80
+
+	ObjectStoreStatusContainerPort = 9090
+	ObjectStoreStatusServicePort   = 9090
+
+	ObjectStoreUIContainerPort = 8080
+	ObjectStoreUIServicePort   = 8080
+
+	ObjectStoreLogLevel = 2
+
+	ObjectStoreStorageClassName = "longhorn-objectstorage"
```

Comment:
Could you point me to the location that we deploy this dummy StorageClass?

Reply:
Sorry, I forgot yesterday to push the changes to the chart:
https://github.com/longhorn/longhorn/blob/fbb2ccfb5f3421276c58af6792f543da72b2fcd4/chart/templates/storageclass.yaml

Reply:
Put a placeholder here to make sure that we don't forget to implement creating `longhorn-objectstorage-static` SC inside Longhorn `kubenetes_configmap_controller`

Reply:
I've implemented that logic in the kubernetes configmap controller.